playwright-tests/tests/wallet-connection.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { ReefKnotService } from '@services';
+import { connectedWalletStorageData, Tags } from '@test-data';
+import { ReefKnotPage } from '@pages';
+import { expect, test } from '@playwright/test';
+import { qase } from 'playwright-qase-reporter';
+import { BrowserService, initBrowserWithWallet } from '@browser';
+
+const wallets = [{ name: 'metamask' }, { name: 'okx' }];
+
+wallets.forEach((wallet) => {
+  test.describe.serial(
+    `ReefKnot. Wallet connection (${wallet.name})`,
+    { tag: [Tags.connectedWallet, `@${wallet.name}`] },
+    async () => {
+      let browserService: BrowserService;
+      let reefKnotService: ReefKnotService;
+      let reefKnotPage: ReefKnotPage;
+
+      test.beforeAll(async () => {
+        ({ browserService, reefKnotService } = await initBrowserWithWallet(
+          wallet.name,
+        ));
+        reefKnotPage = reefKnotService.reefKnotPage;
+        await reefKnotPage.goto();
+        await reefKnotPage.allowUseCookies();
+      });
+
+      test.afterAll(async () => {
+        await reefKnotService.disconnectWalletForce();
+        await browserService.teardown();
+      });
+
+      test(qase(434, 'Connect wallet'), async () => {
+        await qase.parameters({ wallet: wallet.name });
+
+        await test.step('Check the stand appearance before wallet connection', async () => {
+          await expect(
+            reefKnotPage.statsBlock.mainComponent,
+            'Expected the statistic block is not displayed before wallet connection',
```

Comment:
This sentence is not grammatically correct and hard to understand. Please, fix it using Google translate, ChatGPT or something similar. This also applies to other sentences in this file.

Reply:
Thank you. I fixed all assert descriptions in all test files